space-dodge-game/screen-orientation-resize/main.js

```diff
@@ -1,6 +1,11 @@
 document.addEventListener('DOMContentLoaded', function () {
-  if (screen.lockOrientation) {
-    screen.lockOrientation('landscape');
+  // Make the application fullscreen to lock the screen
+  if (document.documentElement.webkitRequestFullScreen) {
```

Comment:
By the way, my testing environment is Android version 5.1 on Nexus 5, and crosswalk-16.44.400.0